seastar.yaml

```diff
@@ -0,0 +1,244 @@
+package:
+  name: seastar
+  epoch: 0
+  version: "25.05.0"
+  description: "Seastar is a high-performance, open-source framework for building C++ applications"
+  copyright:
+    - license: "Apache-2.0"
+  dependencies:
+    runtime:
+      - binutils
+      - boost-dev
+      - c-ares-dev
+      - fmt-dev
+      - gcc
+      - pkgconf
+      - protobuf-dev
+
+environment:
+  contents:
+    packages:
+      - abseil-cpp-dev
+      - bison
+      - boost-dev
+      - build-base
+      - c-ares-dev
+      - doxygen
+      - fmt-dev
+      - gnutls-dev
+      - gtest-dev
+      - hwloc-dev
+      - libsctp-dev
+      - liburing-dev
+      - lz4-dev
+      - numactl-dev
+      - pkgconf-dev
+      - protobuf-dev
+      - python-${{vars.py-version}}
+      - ragel
+      - util-linux-dev
+      - valgrind-dev
+      - xfsprogs-dev
+      - yaml-cpp-dev
+
+vars:
+  py-version: 3.13
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: "https://github.com/scylladb/seastar.git"
+      tag: seastar-${{package.version}}
+      expected-commit: 8df8212e53577e1d8477a5c901457cd61d88afc7
+
+  - uses: cmake/configure
+    with:
+      opts: |
+        -DSeastar_TESTING=OFF \
+        -DPROTOBUF_SOURCE=SYSTEM \
+        -DBUILD_SHARED_LIBS=ON \
+        -DWITH_SSL=system
+
+  - uses: cmake/build
+
+  - uses: cmake/install
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-dev
+    description: ${{package.name}} dev headers
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      runtime:
+        - ${{package.name}}
+        - ${{package.name}}-libs
+    test:
+      pipeline:
+        - uses: test/tw/ldd-check
+
+  - name: ${{package.name}}-libs
+    description: "libraries for ${{package.name}}"
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/usr/lib
+          mv "${{targets.destdir}}"/usr/lib/lib*.so* "${{targets.subpkgdir}}"/usr/lib/
+    test:
+      pipeline:
+        - uses: test/tw/ldd-check
+
```

Comment:
Upstream does provides docs `-DSeastar_DOCS=ON`, should we have a doc subpkg?

Reply:
Even though that build flag exists, the build doesn't actually produce any docs